visidata/guides/MenuGuide.md

```diff
@@ -0,0 +1,24 @@
+# The Menu System
+
+Located at the very top of the screen, the menu organizes many commands within submenu trees.
```

Comment:
"The menu (on the first row of the window) contains most commands, organized within a tree of submenus."